.github/workflows/pre-commit-update.yml

```diff
@@ -0,0 +1,12 @@
+name: Update pre-commit
+
+on:
+  schedule:
+    - cron: "0 20 * * SUN"  # Sunday @ 2000 UTC
+  workflow_dispatch:
+
+jobs:
+  pre-commit-update:
```

Comment:
inb4....I should have linked a repo that doesn't use Towncrier, like [`briefcase-windows-app-template`](https://github.com/beeware/briefcase-windows-app-template/blob/main/.github/workflows/pre-commit-update.yml).

You'll need to pass a few flags to the workflow:
```
  pre-commit-update:
    name: Update pre-commit
    uses: beeware/.github/.github/workflows/pre-commit-update.yml@main
    secrets: inherit
    with:
      pre-commit-source: pre-commit
      create-changenote: false
```

Reply:
Do'h. Knew I forgot something. Thanks for keeping me honest.